source/installation/install-on-windows.txt

```diff
@@ -0,0 +1,99 @@
+.. _c2c-install-windows:
+
+==================
+Install on Windows
+==================
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. |arrow| unicode:: U+27A4
+
+Overview
+--------
+
+Use this tutorial to manually install {+c2c-full-product-name+}
+{+version+} on Windows using a downloaded ``.zip`` zipfile.
+
+.. bug in constant parser DOP-3025
+
+{+c2c-product-name+} Version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-install-past-mongosync.rst
+
+Considerations
+--------------
+
+Consider the following when running {+c2c-full-product-name+} on Windows:
+
+Platform Support
+~~~~~~~~~~~~~~~~
+
+{+c2c-product-name+} is available on the following Windows operating
+systems:
+
+- Windows 10
+- Windows 11
+
+Version Requirements
+~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-mongodb-version.rst
+
+.. bug in constant parser DOP-3025
+
+Install {+c2c-product-name+}
+-------------------------------
+
+Follow these steps to manually install {+c2c-product-name+} from
+the ``.zip``.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Download the zipfile.
+
+      Download the {+c2c-product-name+} zipfile from the following
+      link:
+
+      |arrow| {+mdb-download-center+}
+
+      a. In the :guilabel:`Version` dropdown, select the version of
+         {+c2c-product-name+} to download.
+
+      #. In the :guilabel:`Platform` dropdown, select
+         :guilabel:`Windows`.
+
+      #. In the :guilabel:`Package` dropdown, select
+         :guilabel:`zip`.
+
+      #. Click :guilabel:`Download`.
+
+   .. step:: Extract the files from the downloaded archive.
+
+      Windows 10 builds after 17062  include the ``tar.exe`` command.
+      To extract ``mongosync``, use the ``tar`` command in a system
+      shell:
+
+      .. code-block:: bash
+
+         tar -xf mongosync-*.zip
+
+   .. step:: Ensure the binary is in a directory listed in your ``PATH`` environment variable.
+
+      .. include:: /includes/step-update-path-windows.rst
```

Comment:
[suggestion] 

"The mongosync binary is in the bin/ directory of the unpacked zipfile." For consistency, consider "extracted" rather than "unpacked." Also, as above, consider "ZIP file."